packages/instrumentation-aws-sdk/src/utils.ts

```diff
@@ -28,9 +29,9 @@ export const normalizeV3Request = (
     commandInput: Record<string, any>,
     region: string
 ): NormalizedRequest => {
-    const commandName = toCamelCase(removeSuffixFromStringIfExists(commandNameWithSuffix, 'Command'));
+    const commandName = removeSuffixFromStringIfExists(commandNameWithSuffix, 'Command');
```

Comment:
In my test of a DynamoDB AWS SDK V3 command, I saw that the command was `ListTablesCommand` so we might not need the `toPascalCase()` here?

Reply:
yes looks like we can omit the PascalCase here